kola/tests/rkt/rkt.go

```diff
@@ -63,3 +75,131 @@ func rktEtcd(t cluster.TestCluster) {
 		t.Fatalf("etcd in rkt failed health check: %v", err)
 	}
 }
+
+// we use subtests to improve testing performance here. Creating the aci is
+// more expensive than actually running most of these tests.
+func rktBase(t cluster.TestCluster) {
+	m := t.Machines()[0]
+
+	// TODO this should not be necessary, but is at the time of writing
```

Comment:
Not directly, but I guess the selinux policy we ship is not compatible with the systemd inside stage1 (v222).